src/main/java/com/qubole/utility/RedisCache.java

```diff
@@ -106,10 +106,13 @@ public V getIfPresent(Object o) {
         LOGGER.info("cache hit, key: " + new String(key));
         return valueSerializer.deserialize(reply);
       }
+    } catch (Exception e) {
+      LOGGER.warn("getIfPresent failed for key: " + o + " with exception: " + e.toString());
```

Comment:
```suggestion
      LOGGER.warn("getIfPresent failed for key: " + o + " with exception: ",e);
```
I prefer this because it shows the full stack trace as well.

Reply:
I initially started with full stack trace as well but it dumps too many logs when redis goes down.